types/domhandler/domhandler-tests.ts

```diff
@@ -1,14 +1,14 @@
-import { DomHandler, DomHandlerOptions, Node } from "domhandler";
+import { DomHandler, DomHandlerOptions, Node, DomElement } from "domhandler";
 
-const handler = new DomHandler((error: Error, dom: any) => {
+const handler = new DomHandler((error: Error, dom: DomElement[]) => {
     if (error)
     	console.error('There has been an error...');
     else
         console.log(dom);
 });
 handler.ontext = (data: string) => { console.log(data); };
 handler.onreset = () => { console.log('We have a reset.'); };
-handler.onerror = (error: Error) => { console.error(Error); };
-handler.onopentag = (name: string, attribs) => { console.log(name, attribs); };
+handler.onerror = (error: Error) => { console.error(error); };
+handler.onopentag = (name: string, attribs: { [s: string]: string }) => { console.log(name, attribs); };
```

Comment:
_might_ be cleaner with `Record<string, string>` here, it will resolve to the same thing though...

Reply:
Yes, it would look nicer but would also require TypeScript 2.1. Maybe we can stick with this for now?

Reply:
TypeScript 2.1 have been out over two years now, I think it would be perfectly acceptable to have it as a minimum requirement.

Out of interest, do the tests run with TypeScript 2 and 3?

Reply:
If we add the TypeScript 2.1 requirement then all modules referencing this module also needs to add the same requirement. I think that might be too much of a change right now.